vortex-file/src/read/mod.rs

```diff
@@ -33,6 +34,7 @@ use vortex_ipc::stream_writer::ByteRange;
 pub use crate::read::mask::RowMask;
 
 // Recommended read-size according to the AWS performance guide
+// FIXME(ngates): this is dumb
```

Comment:
Why?

Reply:
It's a hard-coded constant, we don't even know if we're reading from S3 or not